docs/start/multinode.rst

```diff
@@ -31,36 +31,62 @@ Set up multinode ray cluster
 
 .. image:: https://github.com/eric-haibin-lin/verl-community/blob/main/docs/ray/overview.png?raw=true
 
-Submit job to ray cluster
-~~~~~~~~~~~~~~~~~~~~~~~~~
-1. Submit ray job to cluster with the dashboard address you get above.
+Start verl training in ray cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. From ray 2.20, you can directly execute the training scripts in ray head node. Note that the ``trainer.n_gpus_per_node`` should be set to GPU number of single machine and ``trainer.nnodes`` should be set to the total number of your machines. For example:
```

Comment:
ray job submit and run the script in the head is the same thing. The only difference is that ray job submit also packages local folders and submit to the remote